dpctl-capi/source/dpctl_sycl_device_manager.cpp

```diff
@@ -164,6 +164,35 @@ DPCTLDeviceMgr_GetDevices(int device_identifier)
     return wrap(Devices);
 }
 
+int DPCTLDeviceMgr_GetPositionInDevices(__dpctl_keep DPCTLSyclDeviceRef DRef,
+                                        int device_identifier)
+{
+    constexpr int not_found = -1;
+    if (!DRef) {
+        return not_found;
+    }
+
+    const auto &root_devices = device::get_devices();
+    default_selector mRanker;
+    int index = not_found;
+    auto reference_Device = *(unwrap(DRef));
+
+    for (const auto &root_Device : root_devices) {
```

Comment:
Minor nitpick. Please change to proper snake case naming `s/root_Device/root_device`

Reply:
Done :)